files/ko/web/javascript/reference/global_objects/math/floor/index.html

```diff
@@ -16,32 +17,32 @@
 
 
 
-<h2 id="구문">구문</h2>
+<h2 id="Syntax">구문</h2>
```

Comment:
```suggestion
<h2 id="구문">구문</h2>
```

id는 원문과 동일하게 유지하는 것이 아니라 같이 번역해야 합니다.